core/bin/external_node/src/main.rs

```diff
@@ -702,6 +702,17 @@ struct Cli {
     /// Comma-separated list of components to launch.
     #[arg(long, default_value = "all")]
     components: ComponentsToRun,
+    /// Path to the yaml config. If set, it will be used instead of env vars.
+    #[arg(long, requires = "secrets_path")]
+    config_path: Option<std::path::PathBuf>,
+    /// Path to the yaml with secrets. If set, it will be used instead of env vars.
+    #[arg(long, requires = "external_node_config_path")]
```

Comment:
The `requires` attributes look confusing. Can't `config_path` specify that it requires all other paths via multiple `require` attrs?